src/flash-source-buffer.js

```diff
@@ -266,30 +274,27 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
     this.bufferSize_ -= chunk.byteLength;
 
     // base64 encode the bytes
-    let binary = '';
+    let binary = [];
     let length = chunk.byteLength;
 
     for (let i = 0; i < length; i++) {
-      binary += String.fromCharCode(chunk[i]);
+      binary.push(String.fromCharCode(chunk[i]));
     }
-    let b64str = window.btoa(binary);
+    let b64str = window.btoa(binary.join(''));
 
-    // bypass normal ExternalInterface calls and pass xml directly
-    // IE can be slow by default
-    this.mediaSource_.swfObj.CallFunction(
-      '<invoke name="vjs_appendBuffer"' +
-      'returntype="javascript"><arguments><string>' +
-      b64str +
-      '</string></arguments></invoke>');
-
-    // schedule another append if necessary
-    if (this.bufferSize_ !== 0) {
+    /* eslint-disable camelcase */
+    window.vjs_flashEncodedData_ = function() {
```

Comment:
This function (and `vjs_flashEncodeHeader_` above) will collide if there are multiple players on the same page. We need some way to create unique names that we pass into Flash. I suggest the concating with the player id  and random value.